git-hub

```diff
@@ -87,14 +97,142 @@ def push():
             p.stdout.read()
     except FileNotFoundError as e:
         print(e)
         print("This command must be used after git hub pr.")
         sys.exit(1)
 
+def search(keyword, user, comment, number, branch):
+    """Searches open and closed pull request comments for specified keyword.
+    Opens pull-requests.toml file in .git folder to fetch pull requests.
+
+    Parameters
+    ----------
+    keyword : string
+        Searches if any part of user, branch, comment, or number match
+    user : string
+        Search by PR username
+    comment : string
+        Search by PR comment
+    number : string
+        Search by PR number
+    branch : string
+        Search by PR branch
+    """
+    try:
+        open_prs = False
+        closed_prs = False
+        pathToGit = './'
+        while not any(fname == '.git' for fname in os.listdir(pathToGit)):  #if not in directory with .git, keep going back to find file
+            pathToGit = pathToGit+"../"
+        pathToGit = pathToGit+".git/git-hub/pull-requests.toml"
+        f = open(pathToGit,"r")
+        pr_dict = toml.load(f)      #fetches toml file and creates a dictionary
+        open_dict = pr_dict['open pull requests']
+        closed_dict = pr_dict['closed pull requests']
+
+        for key in open_dict.keys():        #iterates through and checks if open pull requsts have keyword
+            pull = open_dict[key]
+            if keyword:         #checks keyword
+                if keyword.upper() in pull.upper() or keyword in str(key):
+                    if not open_prs:
+                        click.echo("OPEN pull requests")
+                        open_prs = True
+                    click.echo('{0}  {1}'.format(key, pull))
+            else:               #checks for all other parameters
+                tempUser, tempBranch = pull.split('/', 1)
+                tempBranch, tempComment = tempBranch.split(': ',1)
+                if number in str(key) and comment.upper() in tempComment.upper() and user.upper() in tempUser.upper() and branch.upper() in tempBranch.upper():
+                    if not open_prs:
+                        click.echo("OPEN pull requests")
+                        open_prs = True
+                    click.echo('{0}  {1}'.format(key, pull))
+
+
+
+        for key in closed_dict.keys():
+            pull = closed_dict[key]
+            if keyword:
+                if keyword.upper() in pull.upper() or keyword in str(key):
+                    if not closed_prs:
+                        if open_prs:
+                            click.echo()
+                            click.echo("--------------------------------")
+                            click.echo()
+                        click.echo("CLOSED pull requests")
+                        closed_prs = True
+                    click.echo('{0}  {1}'.format(key, pull))
+            else:
+                tempUser, tempBranch = pull.split('/', 1)
```

Comment:
Don't repeat code; refactor the few lines above into a function, and re-use here.